salt/modules/vsphere.py

```diff
@@ -3224,6 +3224,19 @@ def vsan_enable(host, username, password, protocol=None, port=None, host_names=N
     return ret
 
 
+def get_dvs_portgroup_assignment(service_instance, host, dvs_name,
+                                 portgroup_name, host_names):
+    '''
+
+    :param service_instance: Reference to the vSphere server
+    :param dvs_name: Name of the DVS
+    :param portgroup_name: Portgroup to examine
+    :param host_names: Names of hosts
+    :return:
```

Comment:
This return description is empty. It also looks like this function doesn't return anything in the code below. Is that intended?